src/System.Security.Principal.Windows/tests/WindowsIdentityTests.cs

```diff
@@ -208,4 +220,21 @@ private static void CheckDispose(WindowsIdentity identity, bool anonymous = fals
             Assert.Throws<ObjectDisposedException>(() => identity.User);
         }
     }
+
+    private static void DoTest(Action<IntPtr> test)
```

Comment:
The names "DoTest" and "test" could be renamed to something more descriptive, for example: 
 test to checkProperties
 DoTest to RetrieveAccessToken